sonar-ws/src/main/java/org/sonarqube/ws/client/project/UpdateVisibilityRequest.java

```diff
@@ -19,26 +19,48 @@
  */
 package org.sonarqube.ws.client.project;
 
-import java.util.Optional;
+import javax.annotation.CheckForNull;
+import javax.annotation.Nullable;
 
 public class UpdateVisibilityRequest {
   private final String project;
-  private final Visibility visibility;
+  private final String visibility;
 
-  public enum Visibility {
```

Comment:
Indeed ! But I'm removing it as it was misleading because the enum was only used in one place. 
And we were tending to not add to much logic in sonar-ws. It can be changed, but here it bothers me as it was only used in one place.